### ISSUE TITLE
Train & Val split

### ISSUE THREAD

**Issue by panmianzhi** [sadmanomee/OOD_Materials_Benchmark#2]
In the paper, it only describes how to split the data into test and train sets, but it doesn't mention how to further divide the training set into train and validation sets. Can you provide guidance on how to split the validation set? Thank you in advance for your help!

**Comment by sadmanomee** [comment]
You can randomly split the training set into further training and validation set.
